packages/wagmi/src/client.ts

```diff
@@ -183,9 +187,16 @@ export class Web3Modal extends Web3ModalScaffold {
       disconnect
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
+    let siweControllerClient: SIWEControllerClient | undefined = undefined
+    if (siweConfig) {
+      siweControllerClient = new Web3ModalSIWEClient({ siweConfig }).client
```

Comment:
SIWE client should not be created here. `@web3modal/siwe` package should export function like `createSiweConfig(...)` that user imports in their dapp and then passes down to wagmi / ethers packages like so (basically same way we pass down wagmi config):

```ts
const wagmiConfig = wagmiConfig(...)
const siweConfig = createSiweConfig(...)
const web3moal = createWeb3Modal({..., siweConfig, wagmiConfig})
```